src/ruby/lib/grpc/generic/client_stub.rb

```diff
@@ -41,7 +41,9 @@ def self.setup_channel(alt_chan, host, creds, channel_args = {})
         channel_args['grpc.primary_user_agent'] += ' '
       end
       channel_args['grpc.primary_user_agent'] += "grpc-ruby/#{VERSION}"
-      unless creds.is_a?(Core::ChannelCredentials) || creds.is_a?(Symbol)
+      unless creds.is_a?(Core::ChannelCredentials) ||
+             creds.is_a?(Core::XdsChannelCredentials) ||
+             creds.is_a?(Symbol)
```

Comment:
nit: can we update this error message to, "creds is not a ChannelCredentials, XdsChannelCredentials, or Symbol"

Reply:
done